docs/src/bot.md

```diff
@@ -0,0 +1,148 @@
+# SnoopCompile Bot
+
+You can use SnoopCompile bot to automatically and continuously create precompile files.
+
+One should add 3 things to a package to make the bot work:
+
+----------------------------------
+
+
+- workflow file:
+
+create a workflow file with this path in your repository `.github/workflows/SnoopCompile.yml` and use the following content:
+
+```yaml
+name: SnoopCompile
+
+on:
+  - push
+
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: ['nightly']
+        julia-arch: [x64]
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.julia-version }}
+      - name: Install dependencies
+        run: julia --project=@. -e 'using Pkg; Pkg.instantiate();'
+      - name : Add SnoopCompile and current package
+        run: julia -e 'using Pkg; Pkg.add(PackageSpec(url = "https://github.com/aminya/SnoopCompile.jl", rev ="packageSnooper")); Pkg.develop(PackageSpec(; path=pwd()));'
```

Comment:
Perhaps use `julia --project` and remove the `Pkg.develop` call.

Reply:
I want the package to be available globally. Because the `snoopiBot` runs another Julia process.